ts/store/reducers/loading.ts

```diff
@@ -5,8 +5,6 @@
  * - ACTION_NAME_(REQUEST|SUCCESS|FAILURE)
  */
 
-import get from "lodash/get";
```

Comment:
do we still need `lodash` ?

Reply:
Not in this file but we still need the package for `lodash/merge` in `ts/theme/index.ts`

Reply:
ok